media/server/ipc/source/SessionManagementServer.cpp

```diff
@@ -25,10 +25,78 @@
 #include "IWebAudioPlayerModuleService.h"
 #include "RialtoServerLogging.h"
 #include <IIpcServerFactory.h>
+#include <grp.h>
+#include <pwd.h>
 #include <sys/stat.h>
+#include <sys/types.h>
+#include <unistd.h>
 
+namespace
+{
+const uid_t kNoOwnerChange = -1; // -1 means chown() won't change the owner
+const gid_t kNoGroupChange = -1; // -1 means chown() won't change the group
+} // namespace
 namespace firebolt::rialto::server::ipc
 {
+static size_t getBufferSizeForPasswordStructureCalls()
+{
+    // Can return -1 on error
+    return sysconf(_SC_GETPW_R_SIZE_MAX);
+}
+
+static uid_t getSocketOwnerId(std::unique_ptr<firebolt::rialto::common::ILinuxWrapper> &linuxWrapper,
```

Comment:
If you pass LinuxWrapper in class constructor, it will be private member attribute. In such case this function and next one (getSocketGroupId) can be private const class member method.